src/carriers/tcpros_carrier/RosSlave.h

```diff
@@ -19,9 +19,12 @@ class RosSlave : public yarp::os::PortReader {
     bool verbose;
     bool worked;
 public:
-    RosSlave(bool verbose) : done(0), verbose(verbose) { 
-        worked = false;
-    }
+    RosSlave(bool verbose) :
+        done(0),
+        portnum(-1),
```

Comment:
This should be causing a warning, because `portnum` is declared before `done`, please invert the order

Reply:
Sure, done https://github.com/robotology/yarp/pull/1412/commits/ef369be0a88ae030f4eb98f243ced2d82173bfc4